### PR TITLE
Fix broken link in README.md

### LINKED ISSUE #288: Broken link in README
### Description

The following link in the README seems broken:

- [Tutorials and Documentation](http://moveit.ros.org/documentation/)

### PR DESCRIPTION
**Pull request by henningkayser**
Fixes #288

### CODE REVIEW

**Review by JafarAbdi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The MoveIt Motion Planning Framework for **ROS 2**. For ROS 1, see [MoveIt 1](ht
 ## General MoveIt Documentation
 
 - [MoveIt Website](http://moveit.ros.org)
-- [Tutorials and Documentation](http://moveit.ros.org/documentation/)
+- [Tutorials](https://ros-planning.github.io/moveit_tutorials/) and [Documentation](http://moveit.ros.org/documentation/source-code-api/)
```
Comment (rhaschke) [inline README.md:14]: Hm. This refers to MoveIT 1 tutorials...

**Review by rhaschke — COMMENTED** [review]
Not sure you want to refer to MoveIt 1 tutorials.

File: README.md (hunk 1 above)
Comment (henningkayser) [reply README.md:14]: The tutorials haven't been ported yet, so I thought this would be better than nothing. I could add a note, though

**Comment by AndyZe** [comment]
This has been fixed by now. It points to https://ros-planning.github.io/moveit_tutorials/

**Pull request closed without merging**